classes/Category.php

```diff
@@ -1489,8 +1489,8 @@ public static function searchByNameAndParentCategoryId($idLang, $categoryName, $
      *
      * @param int $idLang Language ID
      * @param string $path Path of category
-     * @param bool $objectToCreate a category
-     * @param bool $methodToCreate a category
+     * @param object|bool $objectToCreate a category
```

Comment:
```suggestion
     * @param object|false $objectToCreate a category
```

Optional I guess but it would be more correct, I don't think `true` is a valid input here

Reply:
true & false  are not in primitive type for PHPDoc : https://docs.phpdoc.org/3.0/guide/guides/types.html